### ISSUE TITLE
ooxml issue

### ISSUE THREAD

**Issue by alexwennerberg** [zitsen/xlsx2csv.rs#12]
When I try to parse my XLSX sheet, I get the following error:

```
[/Users/awennerb/.cargo/registry/src/github.com-1ecc6299db9ec823/ooxml-0.2.4/src/document/spreadsheet/mod.rs:269] raw = "20210929154340"
[/Users/awennerb/.cargo/registry/src/github.com-1ecc6299db9ec823/ooxml-0.2.4/src/document/spreadsheet/mod.rs:269] style_id = 1
[/Users/awennerb/.cargo/registry/src/github.com-1ecc6299db9ec823/ooxml-0.2.4/src/document/spreadsheet/mod.rs:269] raw = "20211028074817"
[/Users/awennerb/.cargo/registry/src/github.com-1ecc6299db9ec823/ooxml-0.2.4/src/document/spreadsheet/mod.rs:269] style_id = 1
thread 'main' panicked at 'not implemented: cell type not supported: str', /Users/awennerb/.cargo/registry/src/github.com-1ecc6299db9ec823/ooxml-0.2.4/src/document/spreadsheet/worksheet.rs:173:22
```

Reverting to the version before `ooxml` was a dependency resolves this issue. Any idea what the source could be?

**Comment by zitsen** [comment]
I'l check it. if you could upload a demo xlsx, that help a lot.

**Comment by alexwennerberg** [comment]
Sadly the csv contains private data. I will try and reproduce the issue in my free time

Alex

**Comment by St-B** [comment]
I face the same issue, reproducible with the attached xlsx. 
<img width="666" alt="image" src="https://user-images.githubusercontent.com/1963009/161914512-b8053888-0dd4-4ba0-921c-5b2a28ccfee2.png">
[Mappe1.xlsx](https://github.com/zitsen/xlsx2csv.rs/files/8424409/Mappe1.xlsx)

**Comment by zitsen** [comment]
> I face the same issue, reproducible with the attached xlsx. <img alt="image" width="666" src="https://user-images.githubusercontent.com/1963009/161914512-b8053888-0dd4-4ba0-921c-5b2a28ccfee2.png"> [Mappe1.xlsx](https://github.com/zitsen/xlsx2csv.rs/files/8424409/Mappe1.xlsx)

Thanks so much for posting this out, I will work on it.

**Comment by zitsen** [comment]
I've confirmed that this issue is fixed in v0.4.3 , download the prebuilt binary from https://github.com/zitsen/xlsx2csv.rs/releases/tag/v0.4.3 or use `cargo install xlsx2csv -f` to update it.

Thanks again for your contribution, I'm very glad to collaborate with you!
